apple-pie/index.html

```diff
@@ -4,36 +4,49 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- Remember to link your styles -->
+    <link rel="StyleSheet" href="/styles/style.css" type="text/css">
```

Comment:
```suggestion
    <link rel="StyleSheet" href="styles/style.css" type="text/css">
```
Al ponerle la barra le estamos diciendo que se vaya a la raiz del ordenador y eso no es del todo, puedes a no poner la barra o poner "./" pero si le pones las barra el punto es requerido vale??